src/client.js

```diff
@@ -182,6 +183,7 @@ Client.prototype.connect = function(args) {
 			} else {
 				webirc = {
 					password: config.webirc[network.host],
+					username: network.username,
```

Comment:
I think webirc usernames are for the client itself, not to identify the user. Probably would need to update the config format so it takes a username and password instead of just a password. Thoughts on that?